components/Grid/grid.css

```diff
@@ -0,0 +1,39 @@
+.grid {
+  display: grid;
+  grid-template-rows: repeat(3, auto);
+  grid-template-columns: repeat(4, auto);
+  padding: 1em;
+  border: 1px solid var(--warning-color);
+}
+
+.grid-item {
+  background-color: var(--info-color);
+  padding: 3rem;
+  text-align: center;
+  color: var(--tertiary-color);
+  font-weight: var(--font-weight-2);
+}
+.span2 {
+  background-color: var(--warning-color);
+  grid-row: 2/3;
+  grid-column: 2/4;
+}
+.span3 {
+  background-color: var(--error-color);
+  grid-row: 1/4;
+  grid-column: 1/2;
+}
+.span4 {
+  background-color: var(--secondary-color);
+}
+
+.grid-col-2,
+.grid-col-3 {
+  display: grid;
+  gap: 6%;
```

Comment:
I think it's not preferable to use % on gap unit.

Reply:
I'll take a look.. thanks for reviewing